index.html

```diff
@@ -6,6 +6,392 @@
+					<figcaption>
+					<h3>Algorithmic Content</h3>
+					Generating believable messages that don’t feel artificial or templated is key to maintaining the illusion. Tools like <a href="#">Automated Insights Wordsmith</a> hint to possibilities where content can be produced on the fly and personalized to each user. 
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="denial">
+					<img src="assets/icon-processing.png" alt="A chat bubble">
+					<figcaption>
+					<h3>Natural Language Processing</h3>
+					Florence’s conversational style will encourage users to speak to her like a friend. Natural language processing, like those offered by <a href="#">IBM’s Watson</a>, will be needed to extract meaning and respond intelligently. 
+					</figcaption>
+				</figure>
+				<figure class="tagRight" id="cost">
+					<img src="assets/icon-denial.svg" alt="A heart symbol">
+					<figcaption>
+					<h3>Behavioral Modeling</h3>
+					Florence can generate many different messages which, unchecked, could make her a little <strong>too</strong> friendly. A system that learns the preferences of users will be helpful in tailoring which kinds of messages are sent, when, and in what frequency.
+					</figcaption>
+				</figure>
+				<figure class="tagLeft" id="complexity">
+					<img src="assets/icon-integration.png" alt="A chain link">
+					<figcaption>
+					<h3>IoT Integrations</h3>
+					The emergence of connected adherence devices like <a href="#">Proteus</a> pills and <a href="#">Smrxt</a> bottles, combined with the popularity of health devices like <a href="#">Fitbits</a> and <a href="#">Qardio</a>, can provide a more rounded view of patients. Aggregating and synthesizing that information, perhaps using services like <a href="#">Validic</a>, would help Florence interact more appropriately.
+					</figcaption>
+				</figure>
+			</figure>
+		</section>
+	</section>
+	<!-- end behind the scenes -->
+
+	<!-- In Closing -->
+	<section class="row">
+		<h1>In Closing</h1>
+		<article>
+			<p>
+			Our concept, Florence, attempts to address significant causes of non-adherence from a patient’s perspective, and through the use of technology. 
+			</p>
+			<p>
+			The problem of non-adherence, however is broader than those factors. This problem needs to be approached from multiple dimensions: socio-economic, education, healthcare access, and treatment models—to name a few. Technology is just one means of attack, not the only one.  
+			</p>
+			<p>
+			If you’re interested in learning more about this problem, we encourage you to read the <a href="#">World Health Organization’s report Evidence for Action</a>. It provides a digestible overview and deep analysis into the problem.
+			</p>
+			<!-- both elements will be floated left in css -->
+			<img src="assets/book.png">
+
+		</article>
+	</section>
+
+	<footer class="row">
+		<h1>AN EXPLORATION BY <img src="viget-logo.svg" alt="Viget's logo"></h1>
```

Comment:
Yeah, this should not be an `<h1>` and shouldn't be a heading at all.  This can just be a `<p>`.  Please make the logo link to our site too and the alt text can just read "Viget". No need to say it's a logo.